cmd/sidecar/main.go

```diff
@@ -381,6 +384,16 @@ func createBackupHandler(w http.ResponseWriter, req *http.Request) {
 		}
 	}
 
+	if backupConf.ContainerOptions != nil {
+		for _, env := range backupConf.ContainerOptions.Env {
+			if err := os.Setenv(env.Name, env.Value); err != nil {
+				log.Error(err, "failed to set env")
+				http.Error(w, "failed to set env", http.StatusInternalServerError)
+				return
+			}
+		}
+	}
```

Comment:
wouldn't it be better to set these env vars for specific commands? like:
```
xtrabackup := exec.CommandContext(gCtx, "xtrabackup", xtrabackupArgs(string(backupUser), backupPass, &backupConf)...)
env := os.Environ()
for _, e := range backupConf.ContainerOptions.Env {
    env = append(env, fmt.Sprintf("%s=%s", e.Name, e.Value)
}
xtrabackup.Env = env

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/986/commits/ea3f7d52acc5d05556b51f66ef0bdb344119aa53